FrontEnd/src/components/ProfilePage/FormComponents/FormFields/ImageField.js

```diff
@@ -1,39 +1,64 @@
+import classNames from 'classnames';
 import css from './ImageField.module.css';
 
 const ImageField = (props) => {
+    const backgroundImage = {
+        background: `url(${props.value}) lightgray 50% / cover no-repeat`,
+    };
     return (
-        <div className={css['fields__column']}>
+        <div className={classNames(css['fields__column'], {[css['fields__column--logo']] : props.name === 'logo_image'})}>
             <div className={css['fields__label']}>
                 {props.requredField && <label className={css['fields__label--required']}>*</label>}
                 <label
-                    className={`${css['fields__label--text']} ${!props.requredField && css['fields__field--notrequired']}`}
-                >
+                    className={classNames(css['fields__label--text'], {[css['fields__field--notrequired']] : !props.requiredField})}>
                     {props.label}
                 </label>
+                {props.name === 'banner_image' && props.value &&
+                    (<span className={css['upload-file__delete--wrapper']} onClick={() => (props.onDeleteImage(props.name))}>
```

Comment:
Could we use more semantic element for like `button` or at least add role="button"

Reply:
Done